$OpenBSD$

Needs ggrep for -P option but let's not bother with that.
Don't do translations (they're broken).

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -63,7 +63,7 @@ if(EXISTS "${CMAKE_SOURCE_DIR}/.git")
       OUTPUT_STRIP_TRAILING_WHITESPACE
     )
   endif()
-elseif(UNIX AND NOT APPLE)
+elseif(UNIX AND NOT APPLE AND NOT ${CMAKE_SYSTEM_NAME} MATCHES "OpenBSD")
   # Fallback for Ubuntu/Launchpad (extracts Git hash from debian/changelog rather than Git repo)
   # (see https://answers.launchpad.net/launchpad/+question/678556)
   execute_process(COMMAND sh -c "grep -Po '(?<=-)(([a-z0-9])\\w+)(?=\\+)' -m 1 changelog"
@@ -500,5 +500,4 @@ if(UNIX AND NOT APPLE)
   install(FILES packaging/linux/icons/128x128/application-vnd.olive-project.png DESTINATION share/icons/hicolor/128x128/mimetypes)
   install(FILES packaging/linux/icons/256x256/application-vnd.olive-project.png DESTINATION share/icons/hicolor/256x256/mimetypes)
   install(FILES packaging/linux/icons/512x512/application-vnd.olive-project.png DESTINATION share/icons/hicolor/512x512/mimetypes)
-  install(FILES ${OLIVE_QM_FILES} DESTINATION share/olive-editor/ts)
 endif()
